### ISSUE TITLE
Expression measure

### ISSUE THREAD

**Issue by rnaseq** [rnaseq/degraTrans#7]
Select the expression measure(s) to use: e.g. FPKM, counts, TPM? 

Add consideration re. is it easy to extract these from all of the tools?

**Comment by rob-p** [comment]
The only measure above to which I object is FPKM.  If we want to measure relative abundance, we should use TPM (there's no good argument favoring FPKM over TPM).  It may make sense, however, to consider both counts and TPM, as they convey somewhat different information (e.g. how well, or poorly, a method is correcting for feature length).

**Comment by blahah** [comment]
I agree - there is [never](https://github.com/COMBINE-lab/salmon/issues/3) a [reason](http://link.springer.com/article/10.1007%2Fs12064-012-0162-3) to [use FPKM](https://www.youtube.com/watch?v=5NiFibnbE8o) :).

TPM, raw (estimated) counts and perhaps effective counts?

**Comment by mw55309** [comment]
Hi All

Happy to drop FPKM (though I bet you all £5/$5 a reviewer suggests we put it in ;-))

If we are using cufflinks, we need to decide (a) which tool to use to produce the expression measure (cufflinks, cuffquant, cuffnorm) and (b) whether to use (normalised) counts or some form of transformed FPKM.

Can we also explicitly state the formulae for:

TPM calculation from counts
TPM conversion from FPKM (if needed)

**Comment by rnaseq** [comment]
I also agree re. dropping FPKM and using raw counts and TPM measure.

'TPM calculation from counts' and 'TPM conversion from FPKM' are stated in this nice blog post: https://haroldpimentel.wordpress.com/2014/05/08/what-the-fpkm-a-review-rna-seq-expression-units/

**Comment by rob-p** [comment]
@mw55309, I'll take the bet if I pay $ and you pay £ ;P. @Blahah, what is the difference here between "estimated" and "effective" counts?  Are effective counts what you get here when you simply distribute "counts" to transcripts based on nucleotide fraction or NPM (nucleotides-per-million)?
